packages/cashier/src/Components/__tests__/error-dialog.spec.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import { __esModule } from '@babel/preset-typescript';
+import ErrorDialog from '../error-dialog';
+
+jest.mock('Stores/connect', () => ({
+    __esModule: true,
+    default: 'mockedConnect',
+    connect: () => Component => Component,
+}));
+describe('<ErrorDialog />', () => {
+    const props = {
+        confirm_button_text: '',
+        error_message: 'This is the error message.',
+        has_no_close_icon: true,
+        reset: jest.fn(),
+        setShouldShow: jest.fn(),
+        should_show: true,
+        should_not_show_title: '',
```

Comment:
@aswathy-deriv believe that should_not_show_title should be a Boolean type (true/false), not string.
also i believe that there is a mistake in ErrorDialog.propTypes in error-dialog.jsx, and should_not_show_title: PropTypes.string should be PropTypes.bool, because we receive this value from ErrorDialog store and in this store should_not_show_title is equal to false, that's why i believe that we should change:
1) ErrorDialog.propTypes = {
    from should_not_show_title: PropTypes.string to should_not_show_title: PropTypes.bool,
}; in error-dialog.jsx

2) as should_not_show_title is a boolean type, in:
const = props {
        should_not_show_title: false,
      }

Reply:
yes in the error-dialog.jsx component, should_not_show_title is shown as string.i will change to boolean